crenata/utils/datetime.py

```diff
@@ -48,3 +48,13 @@ def to_weekday(date: datetime) -> str:
     days = ["월", "화", "수", "목", "금", "토", "일"]
     day = date.weekday()
     return days[day]
+
+
+def to_relative_date(keyword: str) -> Optional[datetime]:
+    """
+    "내일"과 같은 상대적인 날짜를 datetime으로 변환합니다.
+    """
+    if keyword == "내일":
+        return datetime.now(KST) + timedelta(days=1)
+    else:
+        return None
```

Comment:
```suggestion
    return None
```

다음과 같은 경우에는 else 표현식이 필요없을것 같습니다.

Reply:
4c1b5c65b03ee817cc9b771ef9242c84a0d69b71